client/models.js

```diff
@@ -1,4 +1,6 @@
-import { has, assert, maybeHas, NONE_CATEGORY_ID, stringToColor } from './helpers';
+import { has, assert, maybeHas, NONE_CATEGORY_ID, stringToColor,
+         formatCurrency, getCurrencySymbol, currencyExists } from './helpers';
+import { store } from './store';
```

Comment:
The store imports the models, so there is a cyclic dependency here. We don't ever want that, because that is technically uncertain what is going to happen to imports, in this case ; they might end up being undefined in one file or the other. Can you update your patch to avoid this, please?